python/py-us/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-us
+version             2.0.2
+revision            0
+
+platforms           darwin
```

Comment:
this port doesn't install any architecture-dependent code, so please add `supported_archs noarch` under the `platforms` line

Reply:
Thanks done.